android/java/res/layout/item_fragment_two_line_divider_item.xml

```diff
@@ -3,5 +3,4 @@
     android:layout_width="match_parent"
     android:layout_height=".5dp"
     android:background="#989B9E"
-    android:layout_marginHorizontal="12dp"
-    android:layout_marginVertical="2dp" />
+    android:layout_marginVertical="4dp" />
```

Comment:
Should we have copyright in here as well?

Reply:
We can, though most of the old XML files didn't have any copyright and we started adding copyright in XML from some time ago while creating new files.

Reply:
this is non blocking btw, I'll leave it up to you to decide if necessary but just noticed it.